src/Lean/Meta/Tactic/AC/Main.lean

```diff
@@ -273,7 +273,7 @@ where
 
 inductive ProofStrategy
```

Comment:
We don't need the `ProofStrategy`, we can do it by a single `simp` traversal where you invoke the normalization code from the `Simp.main` `post`.

Reply:
ya, got it. It will also make the `findUnnormalized` easier, because it won't need to deal with multiple operators. Instead, it'll just try to match the head node and check if it's one of our operators. I'll look into it in the next few days, but probably not today.

Reply:
I think you don't need `findUnnormalized`. Your `post` function can check whether the given expression is an AC operator, but the parent is not the same one.

Reply:
done